src/app/app-interactive-map/components/loader/loader.component.ts

```diff
@@ -34,5 +36,12 @@ export class LoaderComponent implements OnInit {
   ngOnInit(): void {
     this.loading = this.store.pipe(select(isLoading));
   }
+
+  ngAfterViewInit(): void {
+    fromEvent(this.refreshButton._elementRef.nativeElement, 'click').pipe(
```

Comment:
Why can we not use normal event binding here?

Reply:
I think it is just because I am thinking about observables all the time hahaha